src/Tools/Extensions.ApiDescription.Client/test/Microsoft.Extensions.ApiDescription.Client.Tests.csproj

```diff
@@ -1,21 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <DefaultItemExcludes>$(DefaultItemExcludes);TestProjects\**\*</DefaultItemExcludes>
     <TestGroupName>ApiDescriptionClientTests</TestGroupName>
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="$(MSBuildThisFileDirectory)..\src\build\**\*" CopyToOutputDirectory="PreserveNewest" LinkBase="build" />
-    <Content Include="$(MSBuildThisFileDirectory)..\src\buildMultiTargeting\**\*" CopyToOutputDirectory="PreserveNewest" LinkBase="buildMultiTargeting" />
-    <Content Include="TestProjects\**\*" CopyToOutputDirectory="PreserveNewest" />
+    <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
+      <_Parameter1>TargetFramework</_Parameter1>
+      <_Parameter2>$(TargetFramework)</_Parameter2>
+    </AssemblyAttribute>
 
-    <Reference Include="Microsoft.Extensions.ApiDescription.Client" />
-  </ItemGroup>
+    <Compile Include="$(SharedSourceRoot)Process\*.cs" LinkBase="ProcessHelpers" />
+    <Compile Include="$(SharedSourceRoot)\CommandLineUtils\Utilities\DotNetMuxer.cs"
+        Link="ProcessHelpers\DotNetMuxer.cs" />
+    <Compile Include="$(ToolSharedSourceRoot)TestHelpers\Temporary*.cs" LinkBase="TestHelpers" />
 
-  <Target Name="CleanTestProjects" BeforeTargets="CoreCompile">
-    <RemoveDir Directories="$(TargetDir)TestProjects" />
-  </Target>
+    <Content Include="..\src\build\**\*" LinkBase="build" />
+    <Content Include="..\src\buildMultiTargeting\**\*" LinkBase="buildMultiTargeting" />
+    <Content Include="TestProjects\**\*" />
 
+    <Reference Include="Microsoft.Extensions.ApiDescription.Client" />
+  </ItemGroup>
```

Comment:
The directory removal was unnecessary ever since we moved bin/ and obj/ folders under artifacts (ages ago) and meant `PreserveNewest` (the default) wastefully acted like `Always`. (About the only case directory removal makes sense is when layouts are changing fairly frequently e.g. in our shared Fx / targeting pack projects. Nothing much here has changed between 2.1.0 and this PR.)

That said, I'll test `clean` operations and make sure `$(TargetDir)TestProjects` isn't left behind.

Reply:
`clean` is working fine and I haven't seen any unexpected side effects of the `RemoveDir` removal